templates/boltzmann/middleware/honeycomb.ts

```diff
@@ -9,77 +10,133 @@ import beeline from 'honeycomb-beeline'
 import isDev from 'are-we-dev'
 void `{% endif %}`;
 
+let otelTracer: OtelTracer | null = null;
+
+function getOtelTracer() {
+  if (!otelTracer) {
+    otelTracer = otelTrace.getTracer('boltzmann', '1.0.0')
```

Comment:
I think this call might be idempotent but I didn't want to chance it.

Reply:
The call's result depends on whether or not the SDK is stood up or not. I sorted this by adding a `get tracer` on the `honeycomb` object.